__tests__/app.test.js

```diff
@@ -199,4 +199,38 @@ describe('GET /api/articles/:article_id and comment count' ,() => {
                 })
         
             })
-        })
\ No newline at end of file
+        })
+        describe('GET /api/articles/:article_id/comments' ,() => {
+            test('status:200, responds with all the comments that match the article_id', () => {   
+                const article_id=1;
+                return request(app)
+                    .get(`/api/articles/${article_id}/comments`)
+                    .expect(200)
+                    .then(({body})=>{
+                        console.log(body.comments, "hsb");
+                        body.comments.forEach((comment) => {expect(comment).toEqual({comment_id: expect.any(Number),author: expect.any(String),created_at: expect.any(String), votes: expect.any(Number),body: expect.any(String),article_id: expect.any(Number) });
+    });
+   
+                                
```

Comment:
Could we please have another test for where a review that exists but has no associated comments? You'll need to think about how to differentiate an array that comes back empty because there's no associated comments vs one that's empty because the review doesn't exist.